uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_contact-actions-menu.scss

```diff
@@ -0,0 +1,70 @@
+.crm_contact-actions {
+  padding: 0;
+  width: 1065px !important;
+  height: auto;
+
+  #bootstrap-theme .row {
+    margin: 0;
+  }
+
+  &::before {
+    background: $gray-light !important;
```

Comment:
this can't be right, why do we need to add a `:before` element to the entire namespace?

Reply:
<img width="829" alt="screen shot 2018-01-19 at 6 34 37 pm" src="https://user-images.githubusercontent.com/6307362/35151711-7f60ef7a-fd47-11e7-8202-e00e49b1a6c0.png">

We need to override this so this style is required.